numpy/lib/tests/test_shape_base.py

```diff
@@ -403,6 +403,14 @@ def test_integer_split_2D_default(self):
         assert_(a.dtype.type is res[-1].dtype.type)
         # perhaps should check higher dimensions
 
```

Comment:
Sorry, forgot about it, but the test needs a decorator similar to:
```
@pytest.mark.skipif(not IS_64BIT, reason="Needs 64bit platform")
```
(the `IS_64BIT` is just from `test_io` and simply `sys.maxsize > 2**32`, otherwise it crashes on 32bit systems.

If you can, it would also be nice if you sqash the test fixups into a single commit (or all into one), but we can also do that when merging. Thanks!

Reply:
@seberg 
Thank you for the tip!

I've commited decorator fix with `--amend`, so all test fixes are in the one commit.

Reply:
Didn't manage to squash the commits, but good enough.